terraform/main.tf

```
@@ -338,6 +338,29 @@ locals {
   ])
 }
 
+resource "aws_s3_bucket_notification" "bucket_notification" {
```

My bad- can we name this according to the corresponding bucket module (since there can only be 1 per bucket)?
```suggestion
resource "aws_s3_bucket_notification" "grants_source_data" {
```

---

yup, fixed